network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/management/NetworkProtectionManagementActivity.kt

```diff
@@ -152,14 +147,10 @@ class NetworkProtectionManagementActivity : DuckDuckGoActivity() {
 
         binding.about.aboutFaq.setClickListener {
             lifecycleScope.launch(dispatcherProvider.io()) {
-                if (subscriptions.isEnabled()) {
-                    globalActivityStarter.start(
-                        this@NetworkProtectionManagementActivity,
-                        WebViewActivityWithParams(url = VPN_HELP_CENTER_URL, screenTitle = getString(R.string.netpFaqTitle)),
-                    )
-                } else {
-                    globalActivityStarter.start(this@NetworkProtectionManagementActivity, NetPFaqsScreenNoParams)
```

Comment:
NIT: NetPFaqsScreenNoParams and NetpFaqActivity can now be removed as well

Reply:
done